src/utils/EIP7412.sol

```diff
@@ -12,12 +12,14 @@ contract EIP7412 {
     /// @param signedOffchainData The data that was returned
     /// from the off-chain interface, signed by the oracle
     function fulfillOracleQuery(
-        address EIP7412Implementer,
+        address payable EIP7412Implementer,
         bytes calldata signedOffchainData
     ) external payable {
         /// @custom:auditor this allows arbitrary calls
         /// to be made to any contract. This may be a security risk.
         /// Please review the contract carefully.
-        IERC7412(EIP7412Implementer).fulfillOracleQuery(signedOffchainData);
+        IERC7412(EIP7412Implementer).fulfillOracleQuery{value: msg.value}(
```

Comment:
**Question:** Why is this now payable?

Reply:
allows the function to forward eth to the `EIP7412Implementer`